### DISCUSSION TITLE
Error with Signing into Opensea using wallet connect

### DISCUSSION THREAD

**Discussion by jonlourie** [WalletConnect/walletconnect-monorepo#3515] [Developer Support]
I am building a mobile wallet with an unverified Project ID when I connect and pair with Opensea and Blur it works but signing messages does not work I get errors like this 

[Unhandled promise rejection: Error: Relay Protocol not supported: undefined]

[Unhandled promise rejection: TypeError: Cannot read property 'signMessage' of undefined]
at utils\EIP155Requests.ts:null in approveEIP155Request

Do I need a verified Project ID or is this issue due to something else. I have been following the Medium tutorial https://medium.com/walletconnect/how-to-build-a-wallet-in-react-native-with-the-web3wallet-sdk-b6f57bf02f9a

**Comment by arthur-zhuk** [comment]
Getting the same error.
